data/json/items/ammo.json

```
@@ -382,7 +382,7 @@
     "symbol": "=",
     "color": "dark_gray",
     "description": "Plastic jacketed copper cable of the type used in small electronics.",
-    "material": [ "plastic", "copper" ],
+    "material": [ { "type": "copper", "portion": 85 }, { "type": "plastic" } ],
```

Are you sure you didn't mean to add a portion value for plastic? As you have it there, it would make the wire 85 parts copper and 1 part plastic.
```suggestion
    "material": [ { "type": "copper", "portion": 85 }, { "type": "plastic", "portion": 15 } ],
```

Feel free to ignore if this is intentional

---

I will be perfectly honest with you - I don't know.
Ever since I started giving out material proportions to items I've been basing it off the electric guitar whose code is
``"material": [
    {
      "type": "wood",
      "portion": 9
    },
    {
      "type": "steel"
    }
  ],``
With the code above, the item is 90% wood and 10% steel. Now, this has not always worked appropriately for me, I admit. The best example being the camp chair which with 975 proportion of steel ends up with... 100% of steel, though I suspect that this is just the game rounding it.
I don't have a perfect idea of how this mechanic works and I have been duplicating code that I have seen work previously.

---

EDIT: Okay, this is a really late wake-up call. Apparently, any time I copied said working code it *didn't work* but the game doesn't show it so I didn't pick it up until I looked at HHG. I'm gonna need to fix up these AND fix up problems I caused in my previous PRs. Thanks for pointing this out, this would have been a terrible cascade effect if I kept on doing this for future PRs.

---

Just keep in mind that by omission, the portion value is 1. And the proportion of a material for an item is basically `(material portion) / (total number of portions)`.

So for your copper wire, the total number of portions is 85 (copper) + 1 (plastic) = 86. Which means it's 85/86 (99%) copper and 1/86 (1%) plastic

---

Yep, I only now realized that. I don't know why this didn't occur to me any time I previously looked at the code for this but I guess mistakes were made. I've got a bunch of cleaning to do now, apologies for the problems.